src/components/GordonLimitedAvailability/index.js

```diff
@@ -0,0 +1,36 @@
+import { Grid, Card, CardContent, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+const GordonLimitedAvailability = ({
+  pageName,
+  backToLocation,
+  backToLink,
+  availableTo,
+  ...OtherProps
```

Comment:
It looks like there's no purpose for the `OtherProps` here. It would only be useful if you were going to pass unspecified props onto a child component, but since you aren't this will not do anything.

Reply:
Removed